### PR TITLE
Fix TF script to compile Tabular Workflows

### PR DESCRIPTION
**Pull request by chmstimoteo**
Fixed TF script to compile tabular workflows pipeline definition without a pre-existing view or table.

# Checklist

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have successfully run the E2E tests, and have included the links to the pipeline runs below
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have updated any relevant documentation to reflect my changes
- [x] I have assigned a reviewer and messaged them

# Pipeline run links:

### CODE REVIEW

File: infrastructure/terraform/README.md [hunk 1]
```diff
@@ -101,11 +100,25 @@ installation.
 
    If you don't have a successful execution from the beginning, re-run until all is deployed successfully.
 
+### Post-Installation Instructions
+
+Now that you have deployed all assets successfully for the 1st time in your Google Cloud Project, the data must flow through all components.
```
Comment (slilichenko) [inline infrastructure/terraform/README.md:105]: Replace "1st" with "first"

File: infrastructure/terraform/README.md [hunk 2]
```diff
@@ -101,11 +100,25 @@ installation.
...
-At this time, the Terraform scripts in this folder create:
+At this time, the Terraform scripts in this folder perform the following tasks:
 
 - Enables the APIs needed
 - IAM bindings needed for the GCP services used
 - Secret in GCP Secret manager for the private GitHub repo
 - Dataform repository connected to the GitHub repo
+- Deploys the data store, feature store, ML pipelines and activation application
+
+It still doesn't deploy the Looker Studio Dashboard.
```
Comment (slilichenko) [inline infrastructure/terraform/README.md:124]: Remove "still" and add a sentence to state that the dashboard installation is a separate step (with the link to the step).

File: infrastructure/terraform/README.md [hunk 3]
```diff
@@ -101,11 +100,25 @@ installation.
 
    If you don't have a successful execution from the beginning, re-run until all is deployed successfully.
 
+### Post-Installation Instructions
+
+Now that you have deployed all assets successfully for the 1st time in your Google Cloud Project, the data must flow through all components. 
+
+At this point in time, you have two options: a) trigger your Cloud Workflow to execute your Dataform worflow. b) wait until the next day 
+when the Cloud Workflow is going to execute according to your schedule. Next, you gonna need to execute the BigQuery stored procedures
```
[the remark this answers is not in the document]
Comment (slilichenko) [reply infrastructure/terraform/README.md:108]: We should give the instructions on how to execute those stored procedures. Shall we wait until we transform the feature store to Dataform?

**Review by slilichenko — COMMENTED** [review]
Need to polish the README a bit.

**Review by slilichenko — APPROVED** [review]

File: infrastructure/terraform/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (chmstimoteo) [reply infrastructure/terraform/README.md]: We have to write a "Basic Operations Guide". Let's bring this topic to our meetings.

**Pull request merged**